src/test/java/org/prebid/server/ApplicationTest.java

```diff
@@ -1718,23 +1723,35 @@ public void optionsRequestShouldRespondWithOriginalPolicyHeaders() {
     }
 
     @Test
-    public void biddersParamsShouldReturnBidderSchemas() throws IOException, JSONException {
+    public void biddersParamsShouldReturnBidderSchemas() throws JSONException {
+        // given
+        final Map<String, JsonNode> bidderNameToSchema = bidderDeps.stream()
+                .map(BidderDeps::getName)
+                .collect(Collectors.toMap(Function.identity(), ApplicationTest::jsonSchemaToJsonNode));
+
+        // when
         final Response response = given(spec)
                 .when()
                 .get("/bidders/params");
 
-        JSONAssert.assertEquals(jsonFrom("params/test-bidder-params-schemas.json"), response.asString(),
-                JSONCompareMode.NON_EXTENSIBLE);
+        // then
+        JSONAssert.assertEquals(bidderNameToSchema.toString(), response.asString(), JSONCompareMode.NON_EXTENSIBLE);
     }
 
     @Test
-    public void infoBiddersShouldReturnRegisteredBidderNames() throws IOException {
-        given(spec)
+    public void infoBiddersShouldReturnRegisteredBidderNames() throws JSONException {
+        // given
+        final List<String> bidderNames = bidderDeps.stream()
```

Comment:
Let's not rely on `BidderDeps` but work with the contents of `static/bidder-params/` directory only instead.